web/init/src/ShipConfigRenderer.jsx

```diff
@@ -33,7 +33,8 @@ export class ShipConfigRenderer extends React.Component {
         <ConfigRender
           fieldsList={groupsList}
           fields={_groups}
-          handleChange={this.props.handleChange}
+          handleChange={this.props.handleChange || (() => { return; })}
```

Comment:
Small nit - you can just do `() => {}` on the right side of `||` for these params

Reply:
`return undefined` will be implied for each of these since it's a empty function

Reply:
that's how it was before the latest update, want to get the code to the same state it was before, going to take some time to test again :)